meetings/agenda-minutes-2021-08-05.md

```diff
@@ -0,0 +1,61 @@
+- JWT: I was going to try to submit a PR to TypeScript.
+- PDL: I don't think we can do anything about this. I don't want to change the behaviour of `valueOf()`.
+- JWT: I had a question about the history of why `valueOf()` throws.
+- PDL: `valueOf()` also allows conversion to numbers in a `+` or `<` operation, for example. This is bad when you mix types, or have non-exact types, and is pretty much always a mistake. `valueOf()` throwing makes you immediately aware that what you're doing is wrong.
+- JWT: Can we document this in the spec? It's currently in the documentation, but the spec text doesn't give any rationale.
+- PFC: Seems like that might be good material for `<emu-note>` in the spec.
+
+### Expanding the charter of IETF to wall-time formats
+- Do we have an opinion about expanding the charter of IETF to more/all Temporal string formats (e.g. PlainDate, PlainTime, etc.)?
+- PDL: Fundamentally yes, but not now. First we need to unblock ourselves for Stage 3. Better compatibility in the future would be good.
+- PFC: Agreed.
+- JGT: I think once we get to the end of adopting the calendar annotation, we'd have a better idea of whether expanding the charter would be likely to be successful, or stall.
+- USA: Agreed. The only gotcha that Neil raised is that Temporal is operating on a technicality. RFC 3339 is exact-time, and we use things from ISO 8601-1 and a few from ISO 8601-2. It could be beneficial to define these wall-time formats.
+- JGT: I think that's a slippery slope and we don't want to get ahead of ourselves and delay the calendar annotation.
+- PDL: I disagree that we'd need to specify that this is allowed, because nowhere in ISO 8601 does it say that it's not allowed.
+- USA: I talked to SFC a few days ago and his opinion was similar.
+
+### IETF discussion about conflicting TZDB versions
+- JGT: Would we be OK if IETF decides that the instant should "win" if there's a conflict with the TZID (e.g. new TZDB since the value was stored)? Background:
+there's some concerns on the IETF list about our current plan to throw exceptions if there's a conflict between the TZID and the offset. I suspect there may be enthusiasm on the list for having the timestamp "win". If that's the consensus of the IETF folks, is this something we could live with? Specifically, it'd mean a normative change to `ZonedDateTime.from()` from `{offset: 'reject'}` to `{offset: 'use'}`. Or we could choose to keep our reject default, but that would mean throwing for values that are accepted by other platforms that follow the new IETF spec. My impression is that the people on the SEDATE list are concerned about ambiguity. I described how our offset option works. One reply was that another option was to not allow any ambiguity at all. I don't agree with that, but I can see where they're coming from.
```

Comment:
```suggestion
there's some concerns on the IETF list about our current plan to throw exceptions if there's a conflict between the TZID and the offset. I suspect there may be enthusiasm on the list for having the timestamp "win", which is unsurprising given that the current RFC 3339 spec is focused on timestamps alone and because prior art in Java's ZonedDateTime API has timestamp winning. Anyway, if "timestamp wins" is the consensus of the IETF folks, is this something we could live with? Specifically, it'd mean a normative change to `ZonedDateTime.from()` from a default of `{offset: 'reject'}` to a default of `{offset: 'use'}`. Or we could choose to keep our `reject` default, but that would mean throwing for values that are accepted by other platforms that follow the new IETF spec. My impression is that the people on the SEDATE list are concerned about ambiguity. I described how our offset option works. One reply was that another option was to not allow any ambiguity at all. I don't agree with that, but I can see where they're coming from.
```